cfpb/consumerfinance.gov
# Remove `header-slug` CSS class

`header-slug__inner` does not have any styles associated with it. 

`header-slug` applies border color styles to the heading that doesn't have a border.

`header-slug` applies a slightly larger top margin on the PfC disclosures tool. I could have replaced this with `u-mt45` to have one-to-one parity, but I think the existing gap width is fine.

## Removals

- Remove `header-slug` class

## How to test this PR

1. The sidebar "my Social Security" and "Get in touch" heading text on http://localhost:8000/consumer-tools/retirement/before-you-claim/about/ should be visually unchanged from https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/about/, as well as https://localhost:8000/consumer-tools/retirement/before-you-claim/about/es/ and https://www.consumerfinance.gov/consumer-tools/retirement/before-you-claim/about/es/
2. The sidebar "Useful resources for new college students" heading http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/about-this-tool/ should be visually unchanged from https://www.consumerfinance.gov/paying-for-college2/understanding-your-financial-aid-offer/about-this-tool/
3. Visit the [disclosures tool](https://[www.consumerfinance.gov](localhost:8000)/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=154022&pid=BAHCA&totl=70950&tuit=14160&hous=7109&book=1150&tran=500&othr=4743&pelg=0&schg=&stag=&othg=&mta=&gib=&wkst=0&ppl=&parl=&perl=&subl=3500&unsl=6000&gpl=&prvl=&prvi=&prvf=&insl=&insi=&inst=&oid=4D0B9265B45716DF4B81EEB4BE117E1DF69871AF) and click through to the third step, and not the "Useful resources for new college students" sidebar is moved up slightly.

## Screenshots

before:

<img width="1065" alt="Screen Shot 2023-03-16 at 2 40 40 PM" src="https://user-images.githubusercontent.com/704760/225723609-d391f3c4-cd1a-4007-ad8e-537a1c566fec.png">

after:

<img width="1078" alt="Screen Shot 2023-03-16 at 2 41 26 PM" src="https://user-images.githubusercontent.com/704760/225723652-c855e5ba-0add-4110-b066-e37e09be55cd.png">
